x/application/types/message_update_param.go

```diff
@@ -14,6 +14,8 @@ func NewMsgUpdateParam(authority string, name string, asType any) *MsgUpdatePara
 	var asTypeIface isMsgUpdateParam_AsType
 
 	switch t := asType.(type) {
+	case uint64:
```

Comment:
Is there a way to enforce that this omissions get caught?

Reply:
You mean type `case` omissions? They should be caught by [`msgUpdateParamSuite`](https://github.com/pokt-network/poktroll/blob/main/tests/integration/params/update_param_test.go#L16) when it attempts to send a `MsgUpdateParam` for a field whose type `case` is missing from the `switch` statement.